cloudferrylib/utils/iterator_as_file.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2014 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+
+from utils import get_log
+
+LOG = get_log(__name__)
+
+
+class IteratorAsFile:
```

Comment:
Please add docstring:
 - General overview (why this class is needed)
 - How, where and when to use it